api/libproject/devstart.rb

```diff
@@ -116,13 +125,11 @@ def dev_up_tanagra(cmd_name, args)
   start_local_db_service()
 
   Dir.chdir('../tanagra-aou-utils') do
-    if op.opts.disable_auth_checks
-      common.status "Starting tanagra-service authorization checks disabled"
-      common.run_inline %W{./run_tanagra_server.sh -a}
-    else
-      common.status "Starting tanagra-service authorization checks enabled"
-      common.run_inline %W{./run_tanagra_server.sh}
-    end
+    common.run_inline("../ui/project.rb tanagra-dep --env local --version #{op.opts.version}")
```

Comment:
Added a project.rb command to load tanagra dependencies. This is located in UI project.rb, but it is referenced from the API project.rb as well.